### DISCUSSION TITLE
2022-06-22 Weekly Meeting 61 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#243] [Weekly Meetings]
Weekly Meeting 61 is happening on Wednesday June 22th, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

## Announcements: 

Lean Coffee? Any demos from the community?

Working Group Updates:
- [OMI Website update](https://github.com/omigroup/website)
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- any others?

**Comment by mrmetaverse** [comment]
Lets review demos and then lean coffee :)

**Comment by mrmetaverse** [comment]
# Engagement with other groups

@ CrewMetaverse 
and 
https://twitter.com/Metaverse_Forum 
and 
XR Guild

**Comment by mrmetaverse** [comment]
Want to contribute content? 

Content themes: `Open` `Metaverse` `Interoperability`

**Comment by mrmetaverse** [comment]
# Scripting 
- Developer Hour Part III was this week!
  - every other week on Tuesdays

**Comment by mrmetaverse** [comment]
# gLTF group 
- aaronfranke#0001 had a question about the collider extension / physics body extension 
- KHR Audio worked on by the group 
  - seeking feedback 
- visit [gLTF Extensions Working Group](https://github.com/omigroup/gltf-extensions) 
- Meets every Thursday at 21:30UTC - 22:30UTC (5:30 - 6:30pm ET)
- contact Fire or Robert

**Comment by mrmetaverse** [comment]
# Media 
- focused on twitter syndication 
- if posting tweets tag open_metaverse related to `#Open` `#Metaverse` `#Interoperability`

**Reply by mrmetaverse** [reply]
enable buttons

**Reply by mrmetaverse** [reply]
#245 created

**Comment by mrmetaverse** [comment]
https://www.figma.com/file/QsTaCznSY63Dn3NJFnUCzw/Untitled?node-id=0%3A1

**Comment by mrmetaverse** [comment]
# Coffee [on figjam](https://www.figma.com/file/QsTaCznSY63Dn3NJFnUCzw/OMI-lean-coffee-6.22.22?node-id=2%3A193)
## Croquet announces their open source Microverse builder
- https://github.com/croquet/microverse
- https://twitter.com/codefrau/status/1539354935430483968

## Engaging with other groups 

> Do we want to engage as a group, or as individuals in the Metaverse Standards Forum? 

![image](https://user-images.githubusercontent.com/63426722/175302417-2c32350e-be4e-40b5-8cce-450f64b5019c.png)


Strengths:   
* Maintain synergy with others
* maintaining independence
* potential alignment with bigger orgs (different angle)
* No charge to participate

Weaknesses:   
* trademark suggests its not open open 

Opportunities:   
* Chance to champion why the Open Metaverse Matters
* Collaboration opps with other groups

Threats: 
* Potential corruption of major players

Motion was made to continue convo next week and take a vote.

**Reply by mikeskydev** [reply]
Join section has since altered the language to discourage individuals from signing up, and should be joined by companies only:

> If you are an individual not affiliated with a member organization, we would welcome keeping you apprised of the latest Forum updates through you signing up to our newsletter.

![image](https://user-images.githubusercontent.com/9121007/175311732-b1bd9fd5-d55e-4cf3-add9-8d31f7b88f7f.png)

**Reply by lyuma** [reply]
It does not limit to only companies: after companies, it also includes "standards organizations, industry associations or universities at no charge".

In my view, it seems reasonable to expect that the chairs on behalf of the OMI [W3C Metaverse Interoperability community group](https://www.w3.org/community/metaverse-interop/) should have standing to join, if we decide to.

**Reply by mikeskydev** [reply]
You're right, I shouldn't have specifically said companies in my original post, but I was focusing on `or as individuals` posed in the original question, which now doesn't seem possible. I do agree that the community group as a whole could (and should) join.
